akka-docs/src/main/paradox/dispatchers.md

```diff
@@ -1,5 +1,17 @@
 # Dispatchers
 
+## Dependency
+
+To use Dispatchers, you must add the following dependency in your project.
```

Comment:
Maybe not so mechanically, but say that dispatchers are part of core akka, which means that they are part of the akka-actro dependency?

Reply:
+1 on this